pages/src/text/lorem-ipsum.ts

```diff
@@ -0,0 +1,91 @@
+import {ESLBaseElement} from '@exadel/esl/modules/esl-base-element/core';
+import {attr} from '@exadel/esl/modules/esl-utils/decorators';
+
+const WORDS = [
```

Comment:
I guess "dictionary" is a more appropriate name.

And it seems to me that if users could import the dictionaries they need and use them, then this would be a very cool feature. Moreover, they could make their own dictionaries and use them.

Reply:
Great idea! I'll take a look how to make this setting available